src/css/app.sass

```diff
@@ -22,3 +22,35 @@ body
 //hide unimplemented ui temporarily
 .temp-hide
     visibility: hidden
+
+.text-headline
+    font-size: 40pt
+    font-family: "ralewaybold"
+
+.text-title
+    font-size: 30pt 
+    font-family: "ralewaybold"
+
+.text-subheader
+    font-size: 20pt
+    font-family: "ralewayregular"
+
+.text-quote
+    font-size: 20pt
+    font-family: "ralewayitalic"
+
+.text-large-body
+    font-size: 14pt
+    font-family: "ralewaybold"
+
+.text-body
+    font-size: 14pt
+    font-family: "ralewayregular"
+
+.text-secondary-body
+    font-size: 12pt
+    font-family: "ralewayregular"
+
+.text-placeholder
+    font-size: 12pt
+    font-family: "ralewayitalic"
```

Comment:
Try to see if you can match the fonts to the existing header classes provided by Quasar instead of defining new classes.
-  Typography classes: https://quasar.dev/style/typography
- Override procedure: https://quasar.dev/style/sass-scss-variables

Reply:
@Th3Doo, I see you changed the classes. Thanks for that, but let's try to use quasar's standard procedure for changing the typography if possible: https://quasar.dev/style/sass-scss-variables

Reply:
Are you wanting me to override the default classes?

Reply:
Jip, but rather override the variables than the classes

Reply:
Here is an example: 
`$h2:        (size: 2.5rem, line-height: 1rem, letter-spacing: normal, weight: 800) !default`